code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm

```diff
@@ -521,7 +521,22 @@
 				H << "<span class='danger'>Your stomach hurts too much as pressure builds up inside of you.</span>"
 				H.adjustToxLoss(6*REM)
 	..()
-
+  
+/datum/chemical_reaction/fartium_explosion
```

Comment:
I swear to god if this meme PR gets merged before the admin logging and creates conflicts I'll be mad.